src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (x) {
```

Comment:
[CODE KNOWLEDGE] - When defining methods inside an object, you can use the method shorthand syntax instead of the `function` keyword. For example:

GOOD EXAMPLE:
add(x) {
  this.result += x;
},